### PR TITLE
docs(adoptions): add references to IDS components extensions and connector gui

### LINKED ISSUE #2404: Adoption Request: EDC IDS-Broker Extension
# Adoption Request as known friend

## General information
Hello,

we as sovity ([Website](https://sovity.de/) | [Github](https://github.com/sovity) | [Linkedin](https://de.linkedin.com/company/sovity)) are working on providing enterprise-ready EDC Connectors and dataspaces as SaaS.

This project, in its first version, can publish assets offered as IDS-resources to a central IDS-Broker, thus establishing initial EDC and IDS-Broker compatibility. We use the event framework of the EDC for this purpose.

| Title | Description | Contact | Links
| :--- | :--- | :--- | :---
| EDC IDS-Broker Extension | EDC connector extension that can register offered assets in form of IDS-resources to a central IDS-Broker. | contact@sovity.de | [link to repo](https://github.com/sovity/edc-extensions)

## Adoption level

Next, please tell us what level of adoption you intend to provide. _(pick only one)_

- [x] Reference a feature as "friend"
- [ ] Incorporate a feature as core EDC component

### LINKED ISSUE #2419: Adoption Request sovity/edc-ui
# Adoption Request

## General information

Hello,

we as sovity ([Website](https://sovity.de) | [Github](https://github.com/sovity) | [Linkedin](https://de.linkedin.com/company/sovity)) are working on providing enterprise-ready EDC Connectors and dataspaces as SaaS. While developing our product we found ourselves working on and improving [eclipse-edc/DataDashboard](https://github.com/eclipse-edc/DataDashboard) to our needs with general usability improvements, better configurability, better asset metadata, form validation and other features for our customers with the big goal of providing production-ready EDC Connectors and dataspaces at ease. 

Our [sovity/edc-ui](https://github.com/sovity/edc-ui) repository encompasses multiple configurations for multiple targets including:
- A variant compatible to an Open Source EDC.
- A variant for the Mobility Data Space (MDS), themed and extended for this particular dataspace (open-source on their request).
- Other configurations that we use to deploy our managed EDCs.

We will keep updating this repository for needs of our products so there are many general improvements, code quality improvements and UX improvements to be expected in the near future.

It will remain one of our goals to provide the default configuration in a compatible way to EDC Connectors built with open source extensions, so this repository might be of interest to other users of the EDC community.

Despite our efforts to keep it functionally compatible, our code will diverge over time, so there are currently no plans to
merge our [sovity/edc-ui](https://github.com/sovity/edc-ui) into [eclipse-edc/DataDashboard](https://github.com/eclipse-edc/DataDashboard) .

Thus we'd love to be able to join in the ranks of "friends" of the EDC Project with this repository to show that there's constant ongoing development on the EDC even outside the sphere of the Eclipse Foundation.

| Title | Description | Contact | Links
| :--- | :--- | :--- | :---
| EDC DataDashboard by sovity (EDC-UI) | DataDashboard extended by sovity. New asset metadata, multiple configurations, form validation and many other features to come. Kept compatible with open-source EDC Connector backends. Apache 2.0 License | contact@sovity.de | [sovity/edc-ui](https://github.com/sovity/edc-ui)

## Adoption level

Next, please tell us what level of adoption you intend to provide. _(pick only one)_

- [x] Reference a feature as "friend"
- [ ] Incorporate a feature as core EDC component

### LINKED ISSUE #2450: Adoption Request: EDC IDS-ClearingHouse Extension
# Adoption Request as known friend

## General information
Hello,

we as sovity ([Website](https://sovity.de/) | [Github](https://github.com/sovity) | [Linkedin](https://de.linkedin.com/company/sovity)) are working on providing enterprise-ready EDC Connectors and dataspaces as SaaS.

This project, in its first version, can log to a central IDS-ClearingHouse, thus establishing initial EDC and IDS-ClearingHouse compatibility. We use the event framework of the EDC for this purpose. 

Following event Use-Cases are logged to the ClearingHouse:
- Confirmed ContractAgreements
- Completed Data-Transferprocesses

| Title | Description | Contact | Links
| :--- | :--- | :--- | :---
| EDC IDS-ClearingHouse Extension | EDC connector extension that can log events to a central IDS-ClearingHouse. | contact@sovity.de | [link to repo](https://github.com/sovity/edc-extensions/tree/main/extensions/clearinghouse)

## Adoption level

Next, please tell us what level of adoption you intend to provide. _(pick only one)_

- [x] Reference a feature as "friend"
- [ ] Incorporate a feature as core EDC component

## More information
There are already two more adoption requests open from us.
- [Adoption Request: EDC IDS-Broker Extension](https://github.com/eclipse-edc/Connector/issues/2404)
- [Adoption Request sovity/edc-ui](https://github.com/eclipse-edc/Connector/issues/2419)

Can be processed together with the other two requests (if still possible, as [PR](https://github.com/eclipse-edc/Connector/pull/2421) already open). We will be happy to answer any questions you may have. We would be very pleased to be included on the list. 

Best regards

### PR DESCRIPTION
**Pull request by juliapampus**
## What this PR changes/adds

Extend known friends list.

## Why it does that

--

## Further notes

--

## Linked Issue(s)

Closes #2404 
Closes #2419
Closes #2450 

## Checklist

- [ ] added appropriate tests?
- [ ] performed checkstyle check locally?
- [ ] added/updated copyright headers?
- [ ] documented public classes/methods?
- [ ] added/updated relevant documentation?
- [ ] assigned appropriate label? (exclude from changelog with label `no-changelog`)
- [ ] formatted title correctly? (_take a look at the [CONTRIBUTING](https://github.com/eclipse-edc/Connector/blob/main/CONTRIBUTING.md#submit-a-pull-request) and [Etiquette for pull requests](https://github.com/eclipse-edc/Connector/blob/main/pr_etiquette.md) for details_)

### CODE REVIEW

**Comment by juliapampus** [comment]
@tmberthold @richardtreier Pls take a look and let me know if your projects are referenced as expected.

**Comment by richardtreier** [comment]
Thank you for the quick processing! 

The UI entry looks good to me.

@tmberthold what about the extension ones?

File: known_friends.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ To get to know how we define "adoptions" and see how to submit a feature, please
...
+| IDS Broker Extension                               | EDC connector extension that can register offered assets in form of IDS Resources to a central IDS Metadata Broker.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | [contact@sovity.de](mailto:contact@sovity.de)                                                                                                | [Link to Repo](https://github.com/sovity/edc-extensions) |
```
Comment (mspiekermann) [inline known_friends.md:10]: Link for the IDS-Metadata Broker extension should be https://github.com/sovity/edc-extensions/tree/main/extensions/broker, right?

In addition most configuration values in the documentation are pointing to instances of the Mobility Data Space. For me it is unclear if you provide an extension to a generic [IDS Metadata Broker](https://github.com/International-Data-Spaces-Association/metadata-broker-open-core) or a MDS flavored version of it. That should be made clear, at least in the title within the table.

File: known_friends.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ To get to know how we define "adoptions" and see how to submit a feature, please
...
+| EDC GUI                                            | EDC Data Dashboard extended by [sovity](https://sovity.de/) and licensed under Apache 2.0. New asset metadata, multiple configurations, form validation, and many other features to come. Kept compatible with open-source EDC Connector interfaces.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | [contact@sovity.de](mailto:contact@sovity.de)                                                                                                | [Link to Repo](https://github.com/sovity/edc-ui) |
```
Comment (mspiekermann) [inline known_friends.md:11]: > New asset metadata, multiple configurations, form validation, and many other features to come. Kept compatible with open-source EDC Connector interfaces.

You are free to describe your solution and its features as the other posts do. But please leave out the marketing statements about possible future features. That's not what the known_friends list is for.

File: known_friends.md (hunk 1 above)
Comment (tmberthold) [reply known_friends.md:10]: Yes, please update the link, the repo has evolved differently than initially thought.
The extension should be compatible with a generic metadata broker.

File: known_friends.md (hunk 2 above)
Comment (tmberthold) [reply known_friends.md:11]: Then probably best to remove the sentence: _New asset metadata, multiple configurations, form validation, and many other features to come._ (@richardtreier ?)

Comment (juliapampus) [reply known_friends.md:11]: Maybe:
```
EDC Data Dashboard customized by [sovity](https://sovity.de/). It integrates the open-source EDC Connector interfaces to manage asstes, set configurations, etc.
```

Comment (richardtreier) [reply known_friends.md:11]: Hmm, I'd prefer something a little more specific such as: 

Maybe:

`EDC Data Dashboard customized by [sovity](https://sovity.de/). It integrates the open-source EDC Connector interfaces while adding opinionated asset fields, adding form validation / design / UX improvements and configuration profiles.`

**Comment by richardtreier** [comment]
@juliapampus 

While the wording is still being revised, could you maybe still add us to the agenda of the upcoming discord call so we could briefly show the open source components we've been working on?

**Comment by juliapampus** [comment]
Some decisions from the committer team (that will be formulated as guidelines by @mspiekermann)
- no company naming in our known friends list
- no column with contact details (details should be provided within the linked repo)
- no promotional phrases (as example, "improvements" could be contributed to the open source repo, if not, it's just an extension/adoption in another repo)

**Comment by juliapampus** [comment]
> @juliapampus
> 
> While the wording is still being revised, could you maybe still add us to the agenda of the upcoming discord call so we could briefly show the open source components we've been working on?

@mspiekermann will contact @SebastianOpriel

**Review by tmberthold — APPROVED** [review]

**Review by richardtreier — APPROVED** [review]

**Review by mspiekermann — APPROVED** [review]

**Pull request merged**